src/meta/model_v2/src/hummock_version_delta.rs

```diff
@@ -12,64 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::collections::HashMap;
-
-use risingwave_pb::hummock::{GroupDelta as PbGroupDelta, HummockVersionDelta};
+use risingwave_pb::hummock::HummockVersionDelta;
 use sea_orm::entity::prelude::*;
 use sea_orm::FromJsonQueryResult;
 use serde::{Deserialize, Serialize};
 
-use crate::{CompactionGroupId, Epoch, HummockSstableObjectId, HummockVersionId};
+use crate::{Epoch, HummockVersionId};
 
-#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize, Default)]
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Default)]
 #[sea_orm(table_name = "hummock_version_delta")]
 pub struct Model {
     #[sea_orm(primary_key, auto_increment = false)]
     pub id: HummockVersionId,
     pub prev_id: HummockVersionId,
-    pub group_deltas: GroupDeltas,
     pub max_committed_epoch: Epoch,
     pub safe_epoch: Epoch,
     pub trivial_move: bool,
```

Comment:
Note that these fields are extracted out just to make the query against sql meta backend more convenience. They are actually duplicated with the information in `FullVersionDelta`.